src/installer/file.rs

```diff
@@ -85,6 +85,16 @@ fn is_elf_file(path: &Path) -> bool {
     check_elf_file(path).unwrap_or(false)
 }
 
+pub fn is_supported_archive(file_name: &str) -> Result<bool, InstallError> {
```

Comment:
Revert all changes to this file since they are not needed. See comments in `download_handler.rs`